etc/integration-tests/src/parse_logs.rs

```diff
@@ -0,0 +1,32 @@
+//! Helper functions for parsing information from nearcore logs.
+
+use crate::ansi_utils;
+use tokio::io::{AsyncBufReadExt, AsyncRead, BufReader};
+
+fn get_height_from_log(line: &str) -> anyhow::Result<u32> {
+    let index = line
+        .find('#')
+        .ok_or_else(|| anyhow::Error::msg("Unknown nearcore log format"))?;
+    let height_str = &line[(index + 1)..(index + 9)];
```

Comment:
I would add some comment why `9`. Or add example of the line which contains block height.
```
2023-08-01T08:26:02.869336Z  INFO stats: #      52 24LGJniHqZjL4nhd55ocngvTofG8jyTKQNCgixcTsJTV Validator | 4 validators 3 peers ⬇ 4.92 kB/s ⬆ 4.80 kB/s 1.90 bps 0 gas/s CPU: 0%, Mem: 59.9 MB
```

Reply:
Done in https://github.com/aurora-is-near/borealis-engine-lib/pull/86/commits/9e5781a2feb0923c0fccbbb50a75a3f3a4a29357